lang/values.md

```diff
@@ -321,9 +321,9 @@ impl PartialOrd for Value {
                 p1 <= p2,
             (Tuple(vals1), Tuple(vals2)) =>
                 vals1 <= vals2,
-            (Variant { idx: idx1, data: data1 }, Variant { idx: idx2, data: data2 }) =>
-                idx == idx1 && data1 <= data2,
-            (Union(chunks1), Union(chunks2)) => chunks1 <= chunks2,
+            (Value::Variant { idx: idx1, data: data1 }, Value::Variant { idx: idx2, data: data2 }) =>
```

Comment:
Just above we use the other `Value` variants so this doesn't seem right.

Can we do `use Value::*;` at the top of the function instead?

Reply:
This should probably not be in this PR anyway, it conceptually seems to belong to your other PR.

Reply:
Yep, this shouldn't be in this PR. good catch!